Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'sprockets-rails', '>= 2.3.3', '< 3'
 gem 'responders', '~> 2.0'
 gem 'roadie-rails', '~> 1.1'
 gem 'x-editable-rails', '~> 1.5.5'
+gem 'foreman-tasks', '>= 0.7.20'
```

Comment:
maybe before the whole bundler dance use sed to remove foreman-tasks from the Gemfile? But it doesn't feel right in any way ;)